packages/askui-nodejs/src/execution/inference-client.ts

```diff
@@ -47,6 +48,11 @@ export class InferenceClient {
   }
 
   async isImageRequired(instruction: string): Promise<boolean> {
+    // V4's image is not required
+    if (this.urls.inference.endsWith('predict')) {
+      return false;
+    }
```

Comment:
Could you clarify why an image isn't necessary? This means that no screenshot will be captured, and the inference request won't include any image.

Reply:
Fixed. Image is always required for the "v4" implementation.